src/android/IBGPlugin.java

```diff
@@ -1133,6 +1135,25 @@ public void setShouldShowSurveysWelcomeScreen(final CallbackContext callbackCont
         }
     }
 
+    /**
+     * Change Locale of Instabug UI elements(defaults to English)
+     *
+     *
+     * @param args .optString(0)      locale name
+     *
+     * @param callbackContext Used when calling back into JavaScript
+     */
+    public void setLocale(final CallbackContext callbackContext, JSONArray args) {
+        String locale = args.optString(0);
```

Comment:
Could args be passed as null? If it's possible, then I'd suggest moving it inside try catch block

Reply:
The parent method already wraps everything in a try/catch

Reply:
buuut.. we already have a try catch block in here :D

Reply:
I'm following the same implementation in the rest of the class, all methods have try/catch blocks inside them. Also other `parse` methods don't.

In any case i think we will have to refactor this in the future to match `argsRegistry` in RN and Flutter